boards/arm/cyclonev_socdk/Kconfig.defconfig

```diff
@@ -28,4 +28,13 @@ config SPI_DW_FIFO_DEPTH
 	default 256
 endif # SPI_DW
 
+if NETWORKING
```

Comment:
The commit message is unrelated to the change; please fix whatever is wrong here